vagrant/packer/scripts/zfs-setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -e
+
+echo '==> Setting up ZFS...'
+
+# It's a VM, ignore fsync(2)
+echo '     * Changing logbias on zroot...'
+/sbin/zfs set logbias=throughput zroot
+
+echo '     * Updating /boot/loader.conf...'
+cat <<'EOF' >> /boot/loader.conf
+vfs.zfs.arc_max="40M"
+vfs.zfs.vdev.cache.size="5M"
+vm.kmem_size="200M"
+vm.kmem_size_max="200M"
```

Comment:
Laptops across the organization thank you.

Reply:
:+1: